### PR TITLE
Docs: troubleshooting guide for RBAC

### PR DESCRIPTION
**Pull request by IevaVasiljeva**
Troubleshooting guide for RBAC.

We're removing the option to disable RBAC for G10, and want to give the users as much information as possible on how to debug and resolve their access control issues.

Resolves https://github.com/grafana/grafana-authnz-team/issues/197

### CODE REVIEW

File: docs/sources/administration/roles-and-permissions/access-control/troubleshooting/index.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+aliases:
+  - ../../../enterprise/access-control/troubleshooting/
+cascade:
```
Comment (vtorosyan) [inline docs/sources/administration/roles-and-permissions/access-control/troubleshooting/index.md:4]: Note that there seem to be some changes in cascading - see [this PR](https://github.com/grafana/writers-toolkit/pull/234/files) updating the docs. We might need to add the target path, as far as I understand to have this working?

**Comment by vtorosyan** [comment]
@grafana/docs-squad tagging you as I remember a conversation where it was being discussed if we should continue maintaining the Troubleshooting pages. 

I am wondering, if this should go directly under RBAC documentation, instead of living in as a separate Troubleshooting guide.

**Comment by vtorosyan** [comment]
@IevaVasiljeva shall we update the [Upgrade guide](https://github.com/grafana/grafana/pull/64927/files#diff-a437cfb0f87d2f40032143832fa624c6ad8140d3922b0e34cf212ac4f97178e0R19) to reference the troubleshooting guide, instead of having the script embedded there?

**Comment by IevaVasiljeva** [comment]
> I am wondering, if this should go directly under RBAC documentation, instead of living in as a separate Troubleshooting guide.

My main reasoning for creating a separate page is that I think it's more discoverable this way (ie, it shows up in the left-hand side content panel). But I'm open to suggestions for other places for this content.

> @IevaVasiljeva shall we update the [Upgrade guide](https://github.com/grafana/grafana/pull/64927/files#diff-a437cfb0f87d2f40032143832fa624c6ad8140d3922b0e34cf212ac4f97178e0R19) to reference the troubleshooting guide, instead of having the script embedded there?

I don't have a strong preference. It's probably easier for users to follow the guide if they don't have to look at another page, so I might slightly lean towards leaving the script in the upgrade guide as well.

**Review by vtorosyan — APPROVED** [review]

**Pull request merged**